templates/docs/base/forms.md

```diff
@@ -64,6 +64,16 @@ To use checkbox component in headings use the `.p-checkbox--heading` variant of
 View example of the heading checkbox components
 </a></div>
 
+#### Indeterminate state checkbox
+
+<span class="p-label--new">New</span>
+
+When a checkbox requires a state between checked and unchecked, `checkbox.indeterminate = true;` can be set via JavaScript, which will cause the checkbox appear in an "indeterminate" state.
+
```

Comment:
I think I would still mention `aria-checked` as an option, especially that we use it in our HTML example.

```suggestion

In cases when JavaScript can't be used for that, set `aria-checked="mixed"` on checkbox to show "indeterminate" state. Please note that it will only change the visual look of the checkbox, it will not affect its state available to the browser via JavaScript.

```